contracts/test/TestACLHelpers.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity 0.4.24;
+
+import "./helpers/Assert.sol";
+import "./helpers/ACLHelper.sol";
+import "../acl/ACLSyntaxSugar.sol";
+import "../acl/ACL.sol";
+
+
+contract TestACLHelpers is ACL, ACLHelper {
+
+    function testEncodeParam() public {
+        Param memory param = Param(2, uint8(Op.EQ), 5294967297);
+
+        uint256 encodedParam = encodeParam(param);
+
+        (uint32 id, uint32 op, uint32 value) = decodeParamsList(encodedParam);
+
+        Assert.equal(uint256(param.id), uint256(id), "Encoded id is not equal");
+        Assert.equal(uint256(param.op), uint256(op), "Encoded op is not equal");
+        Assert.equal(uint256(param.value), uint256(value), "Encoded value is not equal");
+    }
+
+    function testEncodeParams() public {
+        Param[] memory params = new Param[](6);
```

Comment:
Why 6? Only 4 are being used, right? (not a big deal)

Reply:
True! :)